docs/mechanical/Gears.md

```diff
@@ -0,0 +1,219 @@
+
+  
+
+![gears5](images/gears5.png)
+
+  
+
+Helical gears are often the default choice in applications that are suitable for spur gears but have non-parallel shafts. They are also used in applications that require high speeds or high loading. And regardless of the load or speed, they generally provide smoother, quieter operation than spur gears.
+
+  
+
+## Bevel Gear
+
+  
+
+Bevel gears are gears where the axes of the two shafts intersect and the tooth-bearing faces of the gears themselves are conically shaped. Bevel gears are most often mounted on shafts that are 90 degrees apart, but can be designed to work at other angles as well. The pitch surface of bevel gears is a cone.
+
+  
+
+![gears6](images/gears6.png)
+
+  
+
+There are several types of bevel gears based on the shape of their teeth.
+
+  
+
+*   **Straight**
+    *   They have conical pitch surface and teeth are straight and tapering towards apex.
+    *   They are useful to verify the transmission of the motion that is generated between axes that intersect within one same plane, almost always at a 90-degree angle.
+    *   Straight bevel gears have many uses in watches, dentist drills, hand drills and vending machines.
+
+  
+
+*   **Spiral**
+    *   They have curved teeth at an angle allowing tooth contact to be gradual and smooth and operate at very steep planes.
+    *   Spiral bevel gears provide a high level of control over the way in which teeth mesh, and their design allows for certain mounting deflections without excessively increasing the load on either end of the teeth.
+    *   They can be used at high speeds, and are usually employed in motorcycle and bicycle gears.
+
+  
+
+*   **Hypoid**
+    *   These are similar to spiral bevel, but the pitch surfaces are hyberbolic and not conical. The pinion can be offset above or below the gear center, thus allowing larger pinion diameter, longer life, and smoother mesh.
+    *   In addition to being used in industrial machinery, they are commonly used in the automotive industry, where it is used in rear-wheel drive vehicles to connect the driveshaft with the wheels.
+
+  
+
+## Worm Gears
+
+  
+
+Worm gears are constructed of a worm and a gear (sometimes referred to as a worm wheel), with non-parallel, non-intersecting shafts oriented 90 degrees to each other. The worm is analogous to a screw with a V-type thread, and the gear is analogous to a spur gear. The worm is typically the driving component, with the worm’s thread advancing the teeth of the gear.
+
+  
+
+![gears7](images/gears7.png)
+
```

Comment:
Resize

Reply:
How to resize ??

Reply:
Use the <img> tag like you did last time. It has a size(height/width) attribute. Will work now

Reply:
Also, do try building the site locally before sending a PR and see if the content is rendering as it should. Check the contributing guidelines to see how to do this

Reply:
Where was <img> tag used ??

Reply:
\<img\>